ngraph_onnx/onnx_importer/ops_bridge/opset_04.py

```diff
@@ -14,4 +14,6 @@
 # limitations under the License.
 # ******************************************************************************
 
+# Concat-4 supported by opset_03.Concat
```

Comment:
It's actually supported by `opset_01.Concat`, that's where the definition lives.

Reply:
Fixed.